### DISCUSSION TITLE
Recommended way to connect to private git account (hosted on-prem or GitHub enterprise on-prem) and public github account

### DISCUSSION THREAD

**Discussion by manishjain5238** [community/community#59670] [Enterprise]
### Select Topic Area

Question

### Body

Which approach/method is recommended and the more secure way for connecting to a 

- private Git account(on-premises hosted Git or GitHub Enterprise on-premises) - which may or may not be behind a firewall
- public GitHub account (private and public repo)

If I need to call API to get a list of all repositories with commit ids and their content from the above private and public repositories.

**Comment by jsspen** [comment]
Were you able to figured this out?
